### ISSUE TITLE
[UX] "Find user accounts" / "Administer users" should be "Manage user accounts"

### ISSUE THREAD

**Issue by klonos** [backdrop/backdrop-issues#552]
...or alternatively "Manage users" or "Manage people".

We need to change the view label (currently "Administer users"), the menu bar link (currently "Find user accounts") because that page also allows actions to be made on listed accounts, so "manage" is a more fitting word.

Perhaps the page title too for consistency (currently "People").

**Comment by jenlampton** [comment]
Definitely not people. "Manage user accounts" ++

**Comment by quicksketch** [comment]
Closed the PR at https://github.com/backdrop/backdrop/pull/1001. It changed the permission name, rather than just updating the page title. This should be a novice issue I think.

**Comment by quicksketch** [comment]
Thanks @Graham-72 for the PR at https://github.com/backdrop/backdrop/issues/1141!

As I posted over there, I realize that we need an update hook in user.install to update this string in the site's active config directory. So this needs a little additional code.

**Comment by jenlampton** [comment]
Looks like this PR Now has the update hook @quicksketch requested, changing tags to PR NR! :)

**Comment by quicksketch** [comment]
Merged in https://github.com/backdrop/backdrop/pull/1141 into 1.x for 1.3.0. Thanks @Graham-72!

**Comment by klonos** [comment]
Just downloaded a fresh copy of 1.x ...still see it as "Find user accounts" (??)

Manually compared my local dev files to the updated ones from the PR and the commit and they are there. My site status is green (as in no db updates pending) and I have cleared caches multiple times. I don't recall any update with regards to this change running recently though - last one was the one about the setting the default value for the maintenance page cache.

**Comment by quicksketch** [comment]
> Just downloaded a fresh copy of 1.x ...still see it as "Find user accounts" (??)

Hm, the string "Find user accounts" isn't anywhere in the code base any more.

> I don't recall any update with regards to this change running recently though - last one was the one about setting the default value for the maintenance page cache.

It's possible if you have been testing out various different 1.x changes that you ran update.php for some other feature that claimed the update number `user_update_1011()`, so upon visiting update.php, the update from this change may think it has already been run. When this happens to me, I usually update the `system` table for the module that has the conflicting update number and set the `schema_version` back to the previous number so the update will run again. In this case, something like this:

```
UPDATE system SET schema_version = 1010 WHERE name = 'user' AND schema_version = 1011;
```

Then run update.php again and it should pick up the update hook this time.

**Comment by klonos** [comment]
Changing the schema version for ```user``` to 1010 did trigger the pending updates message, but after running the 1011 update script and flushing caches I still see the "Find user accounts". Pointers?

**Comment by Graham-72** [comment]
Strange. The update process has modified a site of mine. Is this where you are seeing it?
![useraccountmenu](https://cloud.githubusercontent.com/assets/10559274/12377223/c7588fd0-bd0d-11e5-801f-118d4a4de5d1.png)

**Comment by klonos** [comment]
Yes:

![backdrop-user_accounts_menu-still_find_user_accounts_instead_of_manage](https://cloud.githubusercontent.com/assets/2423362/12398215/475baa38-be66-11e5-92ee-c739f93b1e6b.png)

**Comment by serundeputy** [comment]
@klonos what does the file `views.view.user_admin` have for the value of `display.page.display_options.menu.title`?

Is it still `Find user accounts`?  if so could be a permissions issue on your `$config['active']` directory or the files in it ... just riffing some possibilities.

**Comment by serundeputy** [comment]
I can verify @klonos I've upgraded 1.2.2 to 1.3.0 and on my local version and on the server the menu item remains `Find user accounts` after running `update.php`.

I checked my active config directory and the json still reflects `Find user accounts`.

Here is the json:

```json
"page": {
833             "display_title": "Page",
834             "display_plugin": "page",
835             "display_options": {
836                 "query": {
837                     "type": "views_query",
838                     "options": []
839                 },
840                 "path": "admin/people/list",
841                 "menu": {
842                     "type": "default tab",
843                     "title": "Find user accounts",
```

I think maybe @Graham-72's `PR` was against an earlier version of `1.x` as in my current `1.3` upgraded site `user_update_1011()` is:

```php
/**
 * Update 'People' menus to 'User accounts'.
 */
function user_update_1011() {
  $config = config('views.view.user_admin');
  if ($config->get('display.default.display_options.title') == 'People') {
    $config->set('display.default.display_options.title', 'User accounts');
  }
  if ($config->get('display.page.display_options.tab_options.title') == 'People') {
    $config->set('display.page.display_options.tab_options.title', 'User accounts');
  }
  $config->save();
}
```

And Graham's from https://github.com/Graham-72/backdrop/blob/c6eaf1dc50e23d580dab928a00d8a5f1c9564094/core/modules/user/user.install

is also schema version 1011:

```php
/**
 * Update 'Find user accounts'  to 'Manage user accounts'
 */
function user_update_1011() {
  $config = config('views.view.user_admin');
	if ($config->get('display.default.display_options.title') == 'User accounts') {
    $config->set('display.default.display_options.title', 'Manage user accounts');
  }
  if ($config->get('display.page.display_options.menu.title') == 'Find user accounts') {
    $config->set('display.page.display_options.menu.title', 'Manage user accounts');
  }
  $config->save();
}
```

So, I think we just need a `PR` to change Graham's update hook from `user_update_1011` to `user_update_1012`  @quicksketch does that seem right to you?

**Comment by serundeputy** [comment]
`PR` passes for PHP 7, but has failures for PHP 5.3, but I don't believe them as they all reference layouts:

```
---- Layout: Layout Interface Test (LayoutInterfaceTest) ----
Status    Group      Filename          Line Function                            
--------------------------------------------------------------------------------
Fail      Other      layout.test        655 LayoutInterfaceTest->testMainConten
    "The &quot;Main page content&quot; block may only be added once to this
    layout." found
Exception Notice     layout.test        661 LayoutInterfaceTest->testMainConten
    Undefined offset: 0
Exception Notice     layout.test        665 LayoutInterfaceTest->testMainConten
    Undefined offset: 0
Fail      Other      layout.test        668 LayoutInterfaceTest->testMainConten
    Found the Save layout button
Fail      Other      layout.test        668 LayoutInterfaceTest->testMainConten
    Found the requested form fields at
Fail      Other      layout.test        669 LayoutInterfaceTest->testMainConten
    Layout saved successfully after removing old content block.
Fail      Other      layout.test        679 LayoutInterfaceTest->testMainConten
    Main content shown in sidebar after moving main content block.
```

**Comment by klonos** [comment]
>@klonos what does the file `views.view.user_admin` (the one in your `$config['active']` directory not the one in the users module)have for the value of `display.page.display_options.menu.title`?

> Is it still `Find user accounts`?

Yep.

Since you can reproduce too, I'm reopening.

**Comment by jenlampton** [comment]
@serundeputy those are random failures.

**Comment by serundeputy** [comment]
new `PR` with the rebase;
https://github.com/backdrop/backdrop/pull/1224

**Comment by quicksketch** [comment]
Thanks @serundeputy! Merged for 1.3.1.

**Comment by klonos** [comment]
Thanx! :+1:
